cdc/scheduler/agent.go

```diff
@@ -272,6 +289,7 @@ func (a *BaseAgent) processOperations(ctx context.Context) error {
 	for tableID, op := range a.tableOperations {
 		switch op.status {
 		case operationReceived:
+			a.logger.Info("Agent start processing operation", zap.Any("op", op))
```

Comment:
Is it acceptable that the operation related log is O(#table) scale.

Reply:
Without this log, it would be difficult to trace scheduling problems.